gc/gc-base/src/test/java/org/projectnessie/gc/base/TestContentBloomFilter.java

```diff
@@ -86,10 +87,12 @@ void testDeltaLakeContent() {
     for (int i = index; i < 10 + index; i++) {
       switch (contentType) {
         case ICEBERG_TABLE:
-          contents.add(IcebergTable.of("temp" + i, i, 42, 42, 42));
+          contents.add(IcebergTable.of(UUID.randomUUID().toString(), "temp" + i, i, 42, 42, 42));
```

Comment:
I understand that the purpose of this change is to prevent misuse of our API, righ?